exclause/with.go

```diff
@@ -76,6 +83,9 @@ func (with With) MergeClause(clause *clause.Clause) {
 		if w.Recursive {
 			with.Recursive = true
 		}
+		if w.Materialized {
+			with.Materialized = true
+		}
```

Comment:
I know I'm repeating myself, but since MATERIALIZED is per-CTE, I don't think we need to have it
in the WITH struct!